openspec/changes/archive/2026-07-18-add-eve-strict-auto-merge-policy/tasks.md

```
@@ -1,64 +1,64 @@
-<!-- Partner DRAFT for GitHub issue #432. Task list for the `add-eve-strict-auto-merge-policy` OpenSpec
-change; enters `Asymmetric-al/core` only via Asymmetric's OpenSpec workflow after sign-off. Acceptance and
+<!-- Accepted implementation task list for GitHub issue #432 and the `add-eve-strict-auto-merge-policy`
+OpenSpec change. Acceptance and
 scope grounded in [VERIFIED-REPO: docs/prds/eve-autonomous-operations/02-implementation-plan.md] and
 [VERIFIED-REPO: docs/prds/eve-autonomous-operations/01-eve-autonomous-operations-platform.md]. -->
 
 ## 1. Define the strict-auto-merge-policy OpenSpec contract
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)